src/pybamm/solvers/processed_variable.py

```diff
@@ -34,11 +36,13 @@ class ProcessedVariable:
 
     def __init__(
         self,
+        variable: str,
         base_variables,
         base_variables_casadi,
         solution,
         time_integral: Optional[pybamm.ProcessedVariableTimeIntegral] = None,
     ):
+        self._variable_name = variable
```

Comment:
We use the word `variable` a lot in this class. Can we change it to `name` throughout instead?
```suggestion
    def __init__(
        self,
        name: str,
        base_variables,
        base_variables_casadi,
        solution,
        time_integral: Optional[pybamm.ProcessedVariableTimeIntegral] = None,
    ):
        self._name = variable
```

Reply:
yup, will do